runtime/tests/checker/access_test.go

```diff
@@ -239,7 +325,15 @@ func TestCheckAccessModifierCompositeVariableFieldDeclaration(t *testing.T) {
 						),
 					)
 
-					assert.NoError(t, err)
+					// private fields in interfaces are invalid
+
+					if isInterface && access == ast.AccessPrivate {
+						errs := ExpectCheckerErrors(t, err, 1)
+
+						assert.IsType(t, &sema.InvalidAccessModifierError{}, errs[0])
```

Comment:
```suggestion
						expectInvalidAccessModifierError(t, err)
```

Reply:
Fixed in f347e69a1af1d42c10180f285eb7e85cda200351